documentation/new-version-quickstart.md

```diff
@@ -172,14 +172,19 @@ var (
 
 ***Write a function to create a resource group***
 ```go
-func createResourceGroup(ctx context.Context, credential azcore.TokenCredential) (armresources.ResourceGroupsCreateOrUpdateResponse, error) {
-	rgClient := armresources.NewResourceGroupsClient(subscriptionId, credential, nil)
+func createResourceGroup(ctx context.Context, credential azcore.TokenCredential) (*armresources.ResourceGroupsClientCreateOrUpdateResponse, error) {
+	rgClient, err := armresources.NewResourceGroupsClient(subscriptionId, credential, nil)
+	if err != nil {
+		return nil, err
+	}
 
-    param := armresources.ResourceGroup{
-        Location: to.StringPtr(location),
-    }
+	param := armresources.ResourceGroup{
+		Location: to.Ptr(location),
+	}
+
+	resp, err := rgClient.CreateOrUpdate(context.Background(), resourceGroupName, param, nil)
```

Comment:
Another case of replace `context.Background()` with `context.TODO()`.  Can be in a separate PR as there might be a more general scrub that needs to happen.